docs/py_docs.rst

```diff
@@ -556,111 +649,88 @@ Raises
 
 .. code-block:: rst
 
    Raises
    ------
-   `IOError`
-       If the file could not be read.
+   IOError
+       Raised if the input file cannot be read.
+   TypeError
+       Raised if parameter ``example`` is an invalid type.
 
-This section should be used judiciously---only for errors that are non-obvious or have a large chance of getting raised.
+Don't wrap each exception's name with backticks, as we do when describing types in :ref:`Parameters <py-docstring-parameters>` and :ref:`Returns <py-docstring-returns>`).
+No namespace prefix is needed when referring to exceptions in the same module as the API.
+Providing the full namespace is often a good idea, though.
 
-.. _py-docstring-see-also:
+The description text is indented by four spaces from the docstring's left justification.
+Like the description fields for :ref:`Parameters <py-docstring-parameters>` and :ref:`Returns <py-docstring-returns>`, the description can consist of multiple paragraphs and lists.
 
-See Also
---------
-
-'See Also' is an optional section used to refer to related code.
-This section can be very useful, but should be used judiciously.
-The goal is to direct users to other functions they may not be aware of, or have easy means of discovering (by looking at the module docstring, for example).
-Routines whose docstrings further explain parameters used by this function are good candidates.
-
-As an example, for a function such as ``numpy.cos``, we would have
+Stylistically, write the first sentence of each description in the form:
 
-.. code-block:: rst
+.. code-block:: text
 
-   See Also
-   --------
-   `sin` : Compute an element-wise Sine function.
-   `tan` : Compute an element-wise Tangent function.
+   Raised if [insert circumstance].
 
-When referring to functions in the same sub-module, no prefix is needed, and the tree is searched upwards for a match.
+.. _py-docstring-see-also:
 
-Prefix objects from other sub-modules appropriately by their greatest common namespace.
-E.g., whilst documenting a ``lsst.afw.tables`` module, refer to a class in ``lsst.afw.detection`` by
+See Also
+--------
 
-.. code-block:: rst
+Use the 'See also' section to link to related APIs that the user may not be aware of, or may not easily discover from other parts of the docstring.
+Here are some good uses of the 'See also' section:
 
-   `afw.detection.Footprint` : Regular detection footprint.
+- If a function wraps another function, you may want to reference the lower-level function.
+- If a function is typically used with another API, you can reference that API.
+- If there is a family of closely related APIs, you might link to others in the family so a user can compare and choose between them easily.
 
-When referring to an entirely different module or package, use the full namespace.
+As an example, for a function such as ``numpy.cos``, we would have:
 
 .. code-block:: rst
 
-   `astropy.table.Tables` : Flexible table data structures
+   See also
+   --------
+   sin
```

Comment:
Descriptions got dropped here; is the implication that they should never be used in "See Also"?

Reply:
The Numpydoc developers originally documented descriptions in the *See also* section, but in reality, those have never been parsed properly.